app/src/androidTest/assets/features/changeContactActivity.feature

```diff
@@ -0,0 +1,43 @@
+Feature: ChangeContact
+  Editing a contact on the change contact page
+
+  Scenario Outline: Change the name of a contact
+      Given I am viewing the change contact activity
+      When I press the contact name
+      And I type "<differentname>"
+      And I close the keyboard
+      And I press on the save button
+      Then the contact should be updated
+      And I should go to the previous activity I visited
+
+      Examples:
+        | differentname  |
+        | OtherName  |
+        | 6j206j206j20  |
+        | @&*#%*$(^(  |
+
+  Scenario Outline: Change the name of a contact to an invalid one
+        Given I am viewing the change contact activity
+        When I press the contact name
+        And I type "<differentname>"
+        And I close the keyboard
+        And I press on the save button
+        And I press confirm on the popup
+        Then I should stay on the page
+
+        Examples:
+          | differentname  |
+          |   |
```

Comment:
Since you have only one example, you can make this a `Scenario` instead of `Scenario Outline`